imports/plugins/core/graphql/server/schemas/currency.graphql

```diff
@@ -23,3 +23,16 @@ type CurrencyEdge implements NodeEdge {
   cursor: ConnectionCursor!
   node: Currency
 }
+
+# QUESTION: What should properties should be included in the CurrencyInput schema?
```

Comment:
We should not have this (at least not needed yet). The prop in the input should be `priceCurrencyCode: String`